earth_enterprise/src/SConstruct

```diff
@@ -350,6 +350,7 @@ installdirs = {
     'server_root': Dir('server/', installdir),
     'server_share': InstServerDir('share'),
     'server_search': InstServerDir('search'),
+    'server_sql': InstServerDir('share/opengee-server/sql'),
 
     'httpd': InstServerDir('gehttpd'),
     'httpdmodules': InstServerDir('gehttpd/modules'),
```

Comment:
'src/installer/products/server/GEServerBase.pm' and 'src/installer/lib/Users.pm' were removed because they were 
obselete or was it part of this PR ?

We would need to ensure that RPM work is in sync with any files removal.

Reply:
It's specified to delete them if possible in the ticket. I checked to make sure the files are not directly referenced on the RPM branch and also tested the installation without them present and saw no issues.

Reply:
ok